doc: README + initial documentation (#51)
grafana/tanka: README.md

# Overview

Tanka is a configuration utility for [Kubernetes](https://kubernetes.io/). It
leverages [Jsonnet](https://jsonnet.org) to allow highly dynamic configuration.

- **:repeat: `ksonnet` drop-in replacement**: Tanka aims to provide the same
  workflow as `ksonnet`: `show`, `diff` and `apply` are just where you expect
  them.

> - **:nut_and_bolt: integrates with the ecosystem**: Tanka does re-invent the

Reviewer: s/does/doesn't/ ?

Author: oops

# Overview

![Tanka Banner](docs/banner.png)

[![Build Status](https://cloud.drone.io/api/badges/grafana/tanka/status.svg)](https://cloud.drone.io/grafana/tanka)
![Golang](https://img.shields.io/badge/language-Go-blue)
![GitHub contributors](https://img.shields.io/github/contributors/grafana/tanka)
![GitHub release](https://img.shields.io/github/release/grafana/tanka)
![License](https://img.shields.io/github/license/grafana/tanka)

Tanka is a configuration utility for [Kubernetes](https://kubernetes.io/). It

> leverages [Jsonnet](https://jsonnet.org) to allow highly dynamic configuration.

Reviewer 2: ```suggestion
Tanka is part a configuration management system for [Kubernetes](https://kubernetes.io/). 
Tanka uses the [Jsonnet](https://jsonnet.org) language to allow flexible, reusable and concise configuration.
```

Reviewer 2: (Looks like the github suggestion didn't quite work, but you get the idea.)

- **:repeat: `ksonnet` drop-in replacement**: Tanka aims to provide the same
  workflow as `ksonnet`: `show`, `diff` and `apply` are just where you expect
  them.
- **:nut_and_bolt: integrates with the ecosystem**: Tanka doesn't re-invent the
  wheel. It rather makes heavy use of what is already there:
  [`jsonnet-bundler`](https://github.com/jsonnet-bundler/jsonnet-bundler) for
  package management and
  [`kubectl`](https://kubernetes.io/docs/reference/kubectl/overview/) for
  communicating with Kubernetes clusters.
- **:hammer: powerful:** Being a `jsonnet`-compatibility layer for Kubernetes,

>   it removes the limitations of static configuration languages.

Reviewer 2: ```suggestion
  it removes the limitations of static (or template-based) configuration languages.
```

> - **:rocket: used in production**: We already use Tanka internally at

Reviewer 2: ```suggestion
- **:rocket: used in production**: We already use Tanka internally at
```

>   Grafana for all of our Kubernetes configuration needs.

Reviewer 2: ```suggestion
  Grafana Labs for all of our Kubernetes configuration needs.
```
